test/Components.feature

```diff
@@ -123,7 +124,7 @@ Feature: Allows dynamic components
             """
         Given an HTML template
             """
-            <my-component color="red" class="rouge" style="font-size:12px;"></my-component>
+            <my-component color="red" class="rouge" style="font-size:12px;">337341200
```

Comment:
What's up with this number and no closing tag?

There's another one on line 163 but I can't comment because it's not part of the commit.